docs/attributes-registry/agent.md

```diff
@@ -0,0 +1,21 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Agent
+
+## Agent Attributes
+
+<!-- semconv registry.agent(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `agent.distro` | string | Agent distribution. [1] | `github.com/signalfx/splunk-otel-collector`; `github.com/aws-observability/aws-otel-collector` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Would be great to at least discuss what the value for this would be for core and contrib

Reply:
I think we can use the github repo values, e.g. `github.com/open-telemetry/opentelemetry-collector-contrib` for contrib. Do you think there are other, better options?

Reply:
I am fine with those, in terms of implementation, would we add the distro value as a field to `component.BuildInfo` or something similar?

Reply:
Should we restrict  the number of examples ( columns)?

Reply:
> I am fine with those, in terms of implementation, would we add the distro value as a field to `component.BuildInfo` or something similar?

Yes, I think so. That's where we put the short commands like "otelcol" today so we can add the corresponding distro string as another field of `BuildInfo` struct.

Reply:
Not resolving to wait on @samiura's comment, but consider my comment resolved here :)